### PR TITLE
Create animal table

### PR DESCRIPTION
**Pull request by edwardoluyara**
- Create a database named `vet_clinic`.
- Create a table named animals with the following columns:
 - `id`: integer
 - `name`: string
 - `date_of_birth`: date
 - `escape_attempts`: integer
 - `neutered`: boolean
 - `weight_kg`: decimal
 
# Include:

- Basic screenshots in the [README.md](https://github.com/oluyaratosin123/Vet-Clinic-Database/blob/feature/animalTable/README.md) file

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+CREATE TABLE dbo.animal
+(
+    id INT NOT NULL PRIMARY KEY, -- primary key column
+    name [NVARCHAR](50) NOT NULL,
+    date_of_birth [NVARCHAR](50) NOT NULL,
+    escape_attempts [NVARCHAR](50) NOT NULL,
+    neutered [NVARCHAR](50) NOT NULL,
+    weight_kg [NVARCHAR](50) NOT NULL
+    -- specify more columns here
```
Comment (t-yanick) [inline schema.sql]: - [ ] Please kindly update your data types to match those specified in the project requirements. See screenshot below for clarifications. 

dataTypes

**Review by t-yanick — APPROVED** [review]
Hi @oluyaratosin123, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql [hunk 2]
```diff
@@ -3,10 +3,10 @@
 CREATE TABLE dbo.animal
 (
     id INT NOT NULL PRIMARY KEY, -- primary key column
-    name [NVARCHAR](50) NOT NULL,
-    date_of_birth [NVARCHAR](50) NOT NULL,
-    escape_attempts [NVARCHAR](50) NOT NULL,
-    neutered [NVARCHAR](50) NOT NULL,
-    weight_kg [NVARCHAR](50) NOT NULL
+    name [NVARCHAR](100) NOT NULL,
```
Comment (Meltrust) [inline schema.sql]: You corrected all data types, but not the `name` on line 6.

According to the requirements:

image

Therefore:

- [ ] As stated by the previous reviewer, please correct your `name` data type.

## Explanation:

NVARCHAR is used only by Oracle/Sybase/MSSQL.   We are using PostgreSQL.

## Hints:

If you want to constraint the number of characters, in PostgreSQL we use `VARCHAR(number of characters allowed)`.

So, in your implementation with a limit of 100 characters you could just use:

`name varchar(100) NOT NULL,`  

...or if you want to allow any number of characters:

`name string NOT NULL,`

Now that you know how to do it, please, kindly implement the change to correct the name column data type. 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @oluyaratosin123,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

You can do it

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @oluyaratosin123,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the correct data type on your `name` column, as requested by the previous review. 💪 🥇 ㊗️ 
## Remember, some syntax varies according to database engines.  Be sure to double-check in the future that the documentation/instructions that you are looking up are for PostgreSQL. ✔️  
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**